src/components/SplashItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { Divider, Icon, Link, Box, useColorModeValue, Center, Text } from "@chakra-ui/react";
+import NextLink from "next/link";
+
+export function SplashItem(props: any) {
```

Comment:
I would probably type the props in an interface. but this is good for now. I'll probably do that in my next PR